src/DotNetWorker.Core/Hosting/WorkerOptions.cs

```diff
@@ -22,12 +23,17 @@ public class WorkerOptions
         /// </summary>
         public InputConverterCollection InputConverters { get; } = new InputConverterCollection();
 
+        /// <summary>
+        /// Gets the worker capabilities.
+        /// </summary>
+        public IDictionary<string, string> Capabilities { get; } = new Dictionary<string, string>();
+
         /// <summary>
         /// Gets and sets the flag for opting in to unwrapping user-code-thrown
         /// exceptions when they are surfaced to the Host. 
         /// </summary>
         public bool EnableUserCodeException { get; set; } = false;
```

Comment:
Since this directly maps to a capability, I suggest updating the get & set to interact with the `Capabilities` dictionary:

``` CSharp
public bool EnableUserCodeException
{
    get => GetBoolCapability(nameof(EnableUserCodeException));
    set => Capabilities[nameof(EnableUserCodeException)] = value.ToString();
}

public bool IncludeEmptyEntriesInMessagePayload
{
    get => GetBoolCapability(nameof(IncludeEmptyEntriesInMessagePayload));
    set => Capabilities[nameof(IncludeEmptyEntriesInMessagePayload)] = value.ToString();
}

private bool GetBoolCapability(string name)
{
    return Capabilities.TryGetValue(name, out string value) && bool.TryParse(value, out bool b) && b;
}
```

Reply:
Nice.